suite-common/wallet-graph/src/graphReducer.ts

```diff
@@ -30,16 +30,18 @@ export type GraphRootState = {
 const updateSectionPoints = (
     state: GraphState,
     payload: {
-        section: 'dashboard' | 'account';
-        points: LineGraphPoint[];
+        section: GraphSection;
```

Comment:
`section` is weird I initially think it's something related to legend of range switcher. What about calling it `graphType` or something like that.

Reply:
03c323a3be9d82529bd3022b1c44ba33a77c0577 - I named it `graphPlacement` because `graphType` evokes something like "line", "bar chart", etc to me...

Reply:
You are right about `type` I don't like `placement` either, that evokes me something like if it's at top or bottom. Maybe something like `graphDataSource`?

Reply:
> You are right about `type` I don't like `placement` either, that evokes me something like if it's at top or bottom. Maybe something like `graphDataSource`?

I really don't know... data source evokes some JSON or XML or other file or just some data representation whcih is taken and put into a graph like its "graph data". .. `graphDisplayLocation` ...? :D

Reply:
I really don't want to imply this has anything to do with display location, it's graph that is just displaying data for Account value or for Portfolio value but you can display it anywhere you want. That's why I think something like `graphDataSource: 'account' | 'portfolio'` is more accurate description.

Reply:
11e4de766fb57bfccbb6ea85dd9d8e4063d9703a